app/views/api/v1/visitor_details/_assignment_detail.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.split_location assignment.split.location
+json.split_name assignment.split.name
```

Comment:
is this an n+1 for the split? should we be doing `includes(:split)` in the controller?